app/models/installed_heat_source.rb

```diff
@@ -0,0 +1,18 @@
+class InstalledHeatSource
+  include Virtus.model
+
+  attribute :key, String
+  attribute :name, String
+  attribute :units, Float
+  attribute :heat_production, Float
```

Comment:
> I am a bit freightened about the capacity term since there's an input_- and output_capacity and @ChaelKruip has a strong opinion about this matter.

Me having a strong opinion about something should not be a motivation to choose a sub-optimal solution here. Do whatever makes more sense code-wise. 

To clarify: my opinion is about the fact that technologies in general have a *set* of capacities:
* output_capacity: the rate at which energy can be deliverd by the technology
* input_capacity: the rate at which energy can be fed into the technology

Without having consistent and physically meaningful names for such basic variables, it will be very difficult to prevent/detect errors made in the energetic calculations. Moreover, adding functionality as described in https://github.com/quintel/etmoses/issues/805 will be a pain.